### DISCUSSION TITLE
Merge 2 pretrained model

### DISCUSSION THREAD

**Discussion by biscowar** [ultralytics/ultralytics#15899] [Q&A]
I have 2 model pretrained. Model one to detect vehicle type and model 2 to detect license plate. Can i merge in only one BEST.PT to run only one yolo istance? If yes how?
Regards

**Comment by glenn-jocher** [comment]
@biscowar merging two pretrained models into a single model isn't directly supported. You can, however, run both models sequentially within the same script to achieve your goal. For more details, please refer to the [Ultralytics documentation](https://docs.ultralytics.com/models/yolov5/).
